react-js/react-part-2.md

```diff
@@ -0,0 +1,175 @@
+# React Part 2
+
+## Projected Time
+
+3 hours 30 mins
+
+- Lesson: 1 hour
+- Guided Practice: 1 hour
+- Independent Practice: 1 hour 30 mins
+
+### Prerequisites
+
+- [React](react.md)
+
+### Motivation
+
+In introduction to [React](/react.md) we discussed about how React makes websites interactive and reduces jQuery code. Now the problem arises if we want to store some information which intent to change and due to change of that information the UI changes i.e the page is re-rendered. Thus, component state is used for such cases.
```

Comment:
```suggestion
In the introduction to [React](/react.md), we discussed how React makes websites interactive and removes the need for jQuery code. 

Now the problem arises: How do we store and change component information we want in the UI?  For example, maybe you want a button's background color to change once it's been clicked.  The solution is to use React `state`.
```

Using jQuery and React together is a big no-no, so let's not give them that idea.